dll/win32/comctl32/imagelist.c

```diff
@@ -4012,6 +4022,23 @@ static inline IImageList2* IImageList2_from_impl(HIMAGELIST himl)
     return NULL;
 }
 
+BOOL is_valid2(HIMAGELIST himl)
+{
+    BOOL valid;
+    __TRY
+    {
+        valid = himl && 
+                himl->IImageList2_iface.lpVtbl == &ImageListImpl_Vtbl && 
```

Comment:
This check makes sure we cannot use v5 lists in a v6 dll

Reply:
Only in the functions that use this change.